systems/data-plan/package/pkg/server/package_test.go

```diff
@@ -0,0 +1,164 @@
+package server
```

Comment:
Same comments as for the Base rates sub system repo tests.

Reply:
addressed